Release notes v0.38.3 (#2541)
grafana/k6: release notes/v0.38.3.md

k6 v0.38.3 is a patch release containing a single fix

### Threshold over already defined sub-metrics will result in an error ([#2538](https://github.com/grafana/k6/issues/2538))

> There was a bug where we were checking if no already added submetrics is added again. Unfortunately we didn't check that this will work with the one submetric we have by default `http_req_duration{expected_response:true}`. After v0.38.0 defining a threshold on it would result in an error.

Reviewer: ```suggestion
There was a bug where we were checking if a submetric had already been added. Unfortunately, we didn't check that this will work with the one submetric we have by default `http_req_duration{expected_response:true}`. After v0.38.0 defining a threshold on it would result in an error.
```

Maybe something like this?

> There was a bug where we were checking if no already added submetrics is added again. Unfortunately we didn't check that this will work with the one submetric we have by default `http_req_duration{expected_response:true}`. After v0.38.0 defining a threshold on it would result in an error.

Reviewer 2: ```suggestion
There was a bug where we were checking if no already added submetrics is added again. Unfortunately, we didn't check that this will work with the one submetric we have by default `http_req_duration{expected_response:true}`. After v0.38.0 defining a threshold on it would result in an error.
```

Author: I went with @olegbespalov suggestion in [5562d91](https://github.com/grafana/k6/pull/2541/commits/5562d919df5c97bebc2dcda1b43a071ea20f00b5)
